integration_test/third_party_apps_data/applications/tomcat/expected_logs.yaml

```diff
@@ -0,0 +1,15 @@
+log_entries:
+- log_name: "cassandra_default_system"
+  field_matchers:
+    jsonPayload.message: "org.apache.catalina.startup.Catalina.start Server startup*"
+    jsonPayload.level: "INFO"
```

Comment:
Oops, missed this. After being extracted, we should remove the severity field from jsonPayload, right?

Reply:
doesn't look like cassandra does, but I'll remove it here.

Reply:
added filter to remove level. need metrics PR to be merged then I'll have integration tests run.